arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -0,0 +1,776 @@
+          },
+          {
+            "name": "createUpdateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccountCreateUpdateParameters"
+            },
+            "description": "The parameters to provide for the current database account."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The database account create or update operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccount"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "DatabaseAccounts_Delete",
+        "description": "Deletes an existing Azure DocumentDB database account.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The database account delete operation will complete asynchronously."
+          },
+          "204": {
+            "description": "The specified account does not exist in the subscription."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/failoverPriorityChange": {
+      "post": {
+        "operationId": "DatabaseAccounts_FailoverPriorityChange",
+        "description": "Changes the failover priority for the Azure DocumentDB database account. A failover priority of 0 indicates a write region. The maximum value for a failover priority = (total number of regions - 1). Failover priority values must be unique for each of the regions in which the database account exists.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "failoverParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FailoverPolicies"
+            },
+            "description": "The new failover policies for the database account."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The failover policy change operation will complete asynchronously.",
```

Comment:
Is this the correct description for OK? It sounds like it belongs to the 202, no?